tools/paas_dashboard/jobs/datadog.rb

```diff
@@ -0,0 +1,46 @@
+require 'dogapi'
+
+SCHEDULER.every '10s', allow_overlapping: false do
+  get_and_emit_data_for_env(
+    service_tag: "service:prod_monitors",
+    data_id_prefix: 'prod'
+  )
+end
+
+SCHEDULER.every '10s', allow_overlapping: false do
+  get_and_emit_data_for_env(
+    service_tag: "service:staging_monitors",
+    data_id_prefix: 'staging'
+  )
+end
+
+SCHEDULER.every '10s', allow_overlapping: false do
+  get_and_emit_data_for_env(
+    service_tag: "service:master_monitors",
+    data_id_prefix: 'ci'
+  )
+end
+
+def get_and_emit_data_for_env(service_tag:, data_id_prefix:)
+  results = get_monitor_results(service_tag)
+  puts results.to_json
+  critical_count, warning_count = get_counts(results)
+  send_event("#{data_id_prefix}_counts", criticals: critical_count, warnings: warning_count)
+end
+
+def get_monitor_results(service_tag)
+  dog.get_all_monitors[1].select { |monitor| monitor['tags'].include?(service_tag) }
```

Comment:
Rather, than filtering the results ourselves, we should probably figure out, how does the ruby package work.

I'm worried about the fact, we gather 68 results, instead of 16. It may be nothing right now, but could be potentially an issue in the future.

FYI, this functionality works as expected in the python library.